src/main/java/net/minecraftforge/client/IItemDecorator.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.client;
+
+import net.minecraft.client.gui.Font;
+import net.minecraft.world.item.ItemStack;
+
+public interface IItemDecorator {
+
+    void render(Font font, ItemStack stack, int xOffset, int yOffset, float blitOffset);
```

Comment:
Please add Javadocs to this interface and its method.
When is this called? What is the expected render state? Are there considerations regarding modifying render state in this method (e.g. enabling/disabling of textures, etc.).